### PR TITLE
Ports "Allows for hostile mobs to perform charge attacks, and adds a giant tackling lobster."

### PR DESCRIPTION
**Pull request by ghost**
# Document the changes in your pull request

Ports "Allows for hostile mobs to perform charge attacks, and adds a giant tackling lobster." from TG.

https://github.com/tgstation/tgstation/pull/52692

# Spriting

image
image

# Changelog

:cl: ArcaneMusic + Kryson on Sprites. Ported by SuperSlayer
rscadd: Lobstrosities!
/:cl:

### CODE REVIEW

**Comment by MusselMain** [comment]
Eyo, this making me hungry for red lobster.

File: code/modules/mob/living/simple_animal/hostile/mining_mobs/lobstrosity.dm [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+	name = "arctic lobstrosity"
+	desc = "A marvel of evolution gone wrong, the frosty ice produces underground lakes where these ill tempered seafood gather. Beware its charge."
+    icon = 'icons/mob/mob.dmi'
+	icon_state = "arctic_lobstrosity"
+	icon_living = "arctic_lobstrosity"
+	icon_dead = "arctic_lobstrosity_dead"
+	mob_biotypes = MOB_ORGANIC|MOB_BEAST
+	mouse_opacity = MOUSE_OPACITY_ICON
+	friendly_verb_continuous = "chitters at"
+	friendly_verb_simple = "chits at"
```
Comment (tattax) [inline code/modules/mob/living/simple_animal/hostile/mining_mobs/lobstrosity.dm]:
```suggestion

```

we don't have these

File: code/modules/mob/living/simple_animal/hostile/mining_mobs/lobstrosity.dm [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+	move_to_delay = 20
+	maxHealth = 150
+	health = 150
+	obj_damage = 15
+	melee_damage_lower = 15
+	melee_damage_upper = 19
+	attack_verb_continuous = "snips"
+	attack_verb_simple = "snip"
+	attack_sound = 'sound/weapons/bite.ogg'
+	weather_immunities = list("snow")
```
Comment (tattax) [inline code/modules/mob/living/simple_animal/hostile/mining_mobs/lobstrosity.dm]:
```suggestion
	weather_immunities = list(WEATHER_SNOW)
```

File: code/modules/mob/living/simple_animal/hostile/mining_mobs/lobstrosity.dm [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+	speak_emote = list("chitters")
+	speed = 3
+	move_to_delay = 20
+	maxHealth = 150
+	health = 150
+	obj_damage = 15
+	melee_damage_lower = 15
+	melee_damage_upper = 19
+	attack_verb_continuous = "snips"
+	attack_verb_simple = "snip"
```
Comment (tattax) [inline code/modules/mob/living/simple_animal/hostile/mining_mobs/lobstrosity.dm]:
```suggestion
	attacktext = "snips"
```

**Comment by KillerOrcaCora** [comment]
Use spriting template correctly, show your sprites.

**Comment by ghost** [comment]
Bruh

**Review by KillerOrcaCora — COMMENTED** [review]
Hungry for red lobster until you realize red lobster sucks now because of pollution...Sprite is fine

**Review by KillerOrcaCora — APPROVED** [review]

**Comment by JamieD1** [comment]
Conflicts

**Pull request closed without merging**